cli/asdf-plugin-manager.sh

```diff
@@ -72,7 +78,11 @@ export_plugins() {
 checkout_plugin_ref() {
     plugin_name="${1}"
     plugin_ref="${2}"
-    git --git-dir "${PLUGINS_REPOS_DIR}/${plugin_name}/.git" checkout "${plugin_ref}" -q
+    git_dir="${PLUGINS_REPOS_DIR}/${plugin_name}"
```

Comment:
btw removing `local` now means you're reassigning a global variable, which these are not. just fyi.

Reply:
It's only a problem if another global var has the same name.
But in general, I will review all functions vars and make them local to avoid any issues in the future.